ethereum/gnosis_protocol_v2/view_solvers.sql

```diff
@@ -36,6 +36,18 @@ known_solver_metadata (address, environment, name) as (
                  (decode('a97DEF4fBCBa3b646dd169Bc2eeE40f0f3fE7771', 'hex'), 'prod', 'BalancerSOR'),
                  (decode('6fa201C3Aff9f1e4897Ed14c7326cF27548d9c35', 'hex'), 'prod', 'Otex'),
                  (decode('e58c68679e7aab8ef83bf37e88b18eb1f6e30e22', 'hex'), 'prod', 'PLM'),
```

Comment:
Should we change the table to include a `deprecated` flag and mark the old addresses as such?

Reply:
We definitely can do this, but that is specifically what the `active` field is already doing.